pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -304,6 +304,7 @@
   how_to_nav_companies: "For companies"
   how_to_nav_afterschool_educators: "For after-school educators"
```

Comment:
do we use this string anywhere else? wondering if we should delete it to keep these i18n files or keep it around, I guess, in case we use it again in the future it would already be translated?

Reply:
I have generally liked to keep a string like this around in case we use it again, but don't feel super strongly either way...